docs/source/multi_gpu.rst

```diff
@@ -678,6 +679,74 @@ Sharded Training can work across all DDP variants by adding the additional ``--p
 
 Internally we re-initialize your optimizers and shard them across your machines and processes. We handle all communication using PyTorch distributed, so no code changes are required.
 
+----------
+
+.. _sequential-parallelism:
+
+Sequential Model Parallelism with Checkpointing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+PyTorch Lightning integration for Sequential Model Parallelism using `FairScale <https://github.com/facebookresearch/fairscale>`_.
+Sequential Model Parallelism splits a sequential module onto multiple GPUs, reducing peak GPU memory requirements substantially.
+We also provide auto-balancing techniques through FairScale, to find optimal balances for the model across GPUs.
+In addition, we use Gradient Checkpointing to reduce GPU memory requirements further, and micro-batches to minimizing device under-utilization automatically.
+
+Reference: https://arxiv.org/abs/1811.06965
+
+.. note:: DDPSequentialPlugin is currently supported only for Pytorch 1.6.
+
+To get started, install all extras using with ``pip install pytorch-lightning["extra"]``
+and Bolts with ``pip install pytorch-lightning-bolts``
```

Comment:
question: Why is bolts needed? or which part?

Reply:
Yeah exactly can we move this to pre-requisites for the example rather than required for the small example?

Reply:
We are going to update this part. I moved Bolts to the example part where it is needed.

Reply:
Can you review the doc again @carmocca @SeanNaren  ?